src/main/java/com/softserve/bookstore/models/User.java

```diff
@@ -15,6 +15,12 @@
 @Getter
 public class User {
 
+    public static final String FIELD_USER_ID = "id_user";
```

Comment:
FYI: this is the way it is done currently at the project we are working on, to have column names in constant strings